base/file.jl

```diff
@@ -616,12 +616,12 @@ function chmod(path::AbstractString, mode::Integer; recursive::Bool=false)
 end
 
 """
-    chown(path::AbstractString, owner::Integer, group::Integer=-1)
+    chown(path::AbstractString, owner::Integer; group::Integer = -1)
 
 Change the owner and/or group of `path` to `owner` and/or `group`. If the value entered for `owner` or `group`
 is `-1` the corresponding ID will not change. Only integer `owner`s and `group`s are currently supported.
```

Comment:
I wonder if we should change this to use `nothing` as the sentinel value instead. We would have to translate `nothing` to -1 for libuv, but using -1 just because libuv does seems to be exposing an implementation detail. So the behavior here would be to error on negative values, replace `nothing` with `-1`, and pass positive integers through unchanged. Not a big deal, just a thought now that we use `nothing` commonly as a sentinel.

Reply:
That makes more sense to me too.